### PR TITLE
Update resource-specific-consent.md

### PR DESCRIPTION
**Pull request by MSFTRickyCastaneda**
Added section on Basic RSC permissions and added a table listing all current basic RSC permissions

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/099254755c52981c4ed1720dca537797e3e77922/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 96 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/11b28b96-3adc-434a-99aa-ad04da9592c9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/graph-api/rsc/resource-specific-consent.md [hunk 1]
```diff
@@ -49,6 +49,14 @@ Use RSC permissions to determine the data access methods for your app. A user's
 
 * **Delegated context RSC permissions (delegated permission)**: Allows an app to access data only on behalf of a signed-in user. No access is allowed in the absence of a signed-in user. Only authorized users can install an app in a specific scope. They can also grant any delegated RSC permissions that the app requests in that specific scope at app installation. For example, if regular members have the permission to install an app inside a team, then they can also grant delegated RSC permission to the app in that specific team.
 
+### Basic RSC Permissions
+
+A limited set of RSC permissions have been reviewed by Microsoft privacy and security teams and have been deemed low risk. These permissions can be consented to at all times by any user when they install and use an app. The following low-risk basic permissions will always be consented to upon installation. 
+
+| Permission name | Action | Type: Delegated | Type: Application |
+| ----- | ----- | :-----: | :-----: |
+|`TeamsActivity.Send.Group`|Send activity feed notifications to users in this team. | NA | Supported |
```
Comment (subray2014) [inline msteams-platform/graph-api/rsc/resource-specific-consent.md:58]: >Group [](http://example.com/codeflow?start=21&length=5)

What about .chat and .user?

**Review by subray2014 — APPROVED** [review]
:shipit:

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/15bd6466a35eeccc575dcae30a6cada7f9c02f42/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 96 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e40cff75-12b5-4ac7-bbe3-a64a99825f98) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Pull request merged**